test-e2e/utils/steps_file.ts

```diff
@@ -113,6 +200,29 @@ module.exports = function() {
       // eslint-disable-next-line no-console
       assert.equal(await this.executeScript(() => typeof jwplayer === 'undefined' ? undefined : jwplayer().getState),
           undefined);
+    },
+    isMobile: async function(this: CodeceptJS.I, isMobile?: boolean) {
+      if (isMobile !== undefined) {
```

Comment:
Same here as above, maybe we can have `if(!isMobile) {...}` instead of `isMobile !== undefined` 🙂

Reply:
This code is also unused. I think the intention was to provide a short circuit if the input was defined (true or false) to not repeat the check, so callers could kind of just call it with a potentially existing variable in-line, but it's really confusing.  Thanks for flagging this, I'll just remove the code.